content/actions/using-workflows/caching-dependencies-to-speed-up-workflows.md

```diff
@@ -369,7 +371,7 @@ jobs:
         env:
           GH_TOKEN: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
           REPO: {% raw %}${{ github.repository }}{% endraw %}
-          BRANCH: refs/pull/{% raw %}${{ github.event.pull_request.number }}{% endraw %}/merge
+          BRANCH: {% raw %}${{ github.ref_name }}{% endraw %}
```

Comment:
Do you think this would work @Mogyuchi?

```suggestion
          BRANCH: {% raw %}${{ github.head_ref }}{% endraw %}
```

When I used the existing code, `gh actions-cache` didn't return anything for me

Reply:
@corneliusroemer 
If you have tried with workflow_dispatch, then your perception that it does not work is correct. This is because `github.head_ref` is an empty string in the workflow_dispatch event.